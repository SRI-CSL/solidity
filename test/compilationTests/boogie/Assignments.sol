pragma solidity ^0.4.17;

contract LocalVars {
    uint contractVar;

    function doSomething(uint param) public returns (uint) {
        uint localVar = 5;
        localVar += param;
        contractVar += localVar;
        return contractVar;
    }
}