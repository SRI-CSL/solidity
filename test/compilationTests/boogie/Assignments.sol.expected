Verifying Assignments::__verifier_main: OK
No errors found.
